hack/verify_history.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
```

Comment:
How does this script correlate with `Generate/check CMA CSV files`? Consider moving the addition of this file to a separate commit (or possible PR).

Reply:
Ah yes, it's not related. I have as a goal to have few commits so that future rebases won't have to manage a lot of commits, but it's probably better to split this one out as you mention.

Reply:
I have split it into a separate commit.